packages/daimo-api/src/api/profile.ts

```diff
@@ -134,6 +135,10 @@ export class ProfileCache {
     return this.linkedAccounts.get(addr) || [];
   }
 
+  getProfilePicture(addr: Address) {
+    return `${daimoDomainAddress}/profile/${addr}/pfp.png`;
```

Comment:
hm I see `{ type: "image/jpeg" }` above, is this a typo?

Reply:
Yes, it is. Thanks. Fixed in the latest commit